frontend/src/pages/LogsPage/LogsTable/LogDetails.tsx

```diff
@@ -106,6 +106,7 @@ export const LogDetails: React.FC<Props> = ({
 		source,
 		service_name: serviceName,
 		service_version: serviceVersion,
+		environment: null, // TODO: replace this when implementing for logs
```

Comment:
just confirming this todo is for the follow up PR which will add querying for the environment value?